ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_extrema_finding.py

```diff
@@ -344,3 +344,44 @@ def test_numpy_fmax(
         out=None,
         where=where,
     )
+
+# fmin
```

Comment:
The test for your function is failing with an index error `IndexError: list index out of range`,kindly look into it and do let me know if you have any other questions :slightly_smiling_face: 
You can look into the failing logs from here too https://github.com/unifyai/ivy/actions/runs/4589665980/jobs/8104717874?pr=13625

Reply:
can u suggest something what to do, as this issue is created by fmax function and not with the fmin function I added.

Reply:
i think there is a problem with the action workflows of ivy